ios/native.xcodeproj/project.pbxproj

```diff
@@ -320,8 +320,10 @@
 			developmentRegion = English;
 			hasScannedForEncodings = 0;
 			knownRegions = (
+				English,
 				en,
 				Base,
+				"he-IL",
```

Comment:
Not sure why this changed (cc @georgegillams)

Reply:
Looks fine to me 👍